charts/incubator/frigate/values.yaml

```diff
@@ -17,17 +17,18 @@ global:
 # -- In most cases the "enabled" key is only used internally to "render" or not the configuration
 # - Some parts of the config bellow are slightly modified so they can be added on SCALE UI. Mainly lists.
 # - Do not blindly copy paste configuration from upstream.
+# - All of the frigate app config is in the questions, so we don't need to default any of it here
```

Comment:
No please. Normal Helm users will have no idea what values are expected without defaults.
Also CI uses these values. 

Questions override these values. 

render_config is false to all of them anyway.

Reply:
Okay, that makes sense for the helm users. We need to keep values.yaml in good shape regardless of the questions.

The original main benefit to users of this PR is removing the defaults so that inheriting global level settings to the camera level works. Now I took a closer look at the settings and how they're grouped under render_config options and I don't see any major cases where the defaulting gets in the way of inheriting a useful global setting. Obviously the defaults block some cases, but not any that seem like users would commonly want to do it. I originally thought this was a bigger annoyance, but maybe not after a closer look. 

Not sure exactly when I'll get back to this, but I have a couple things I want to try out (specifically, why does the defaults for the strings from values.yaml not come through when the integers do? Is it because of the null:true used on many of the integer questions? Can we get the integers to behave the same way?). **As it is right now, I'll probably back out 80% of this PR and just keep the changes that fix what is required vs optional.**


Other idea: Is it okay to just comment out most of the frigate config lines in values.yaml? I still think there is value in removing excess defaults to make inheriting settings easier, even though I don't think it's as critical as I used to. As someone with basically no helm experience, this seems like a fine workaround to me, but I don't really know best practices.

Reply:
So lets start of how the UI works.
When a field is under a show_subquestions_if: EXPRESSION, and the expression evaluates to false.
None of the values under `subquestions` are taken into account. In that case, the values from `values.yaml` are used.

When the expression evaluates to true, all subquestions will override the values from values.yaml. 
(That's why we also set the same defaults to questions.yaml and values.yaml. 
If we didn't have the default values in questions.yaml, and someone ticks render_config will have all the subquestions fields empty, and it will end up empty in the configmap too.)

I'm fine with commenting out values.yaml configuration, but still needs to take care of the questions.yaml and their defaults.
Which most likely means also adapting the configmap. Adapting the configmap (eg to only take into account values when render_config is true). questions,.yaml will also override any values from values.yaml. So no point to commenting out values.yaml :P

---

From a quick look, looks like you adapted the configmap to use more `render_config` bools.
Only "nitpick" i found was the `with` on booleans, which won't work as you might think. 
It's just a glorified `if` + changing the `.` to point on the value it checked.

So `with $false_value` will never enter the block.

Reply:
As for which config has to have default or not, I'm pretty sure you are way more experienced with.
I was only going by reading the configuration file. I haven't even tried to deploy it at all.

So on this side, I'll have to take your word.